### PR TITLE
first draft

### PR DESCRIPTION
**Pull request by guyardito**

### CODE REVIEW

File: MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift [hunk 1]
```diff
@@ -8,29 +8,129 @@
...
+
+// Given....
+let x = 5.0
+let y = 12
+let a = 321
+let b = 32
+
+
+/*: Question 1
+### 1. Print "true" if a modulo b is zero
```
Comment (JimCampagno) [inline MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift]: Similar change to the prior lab. Adding the "is equal to zero part" so it reads as.

Print "true" if a modulo b is equal to zero.

File: MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift [hunk 2]
```diff
@@ -8,29 +8,129 @@
...
+*/
 // write your code here
+switch a % b {
+case 0:
+	print("true")
+	
+default:
+	// do nothing
+	break
+}
```
Comment (JimCampagno) [inline MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift]: Need to remove the answer.

File: MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift [hunk 3]
```diff
@@ -8,29 +8,129 @@
...
+### 2. Print "true" if y divided by x is greater than three, otherwise print false
+*/
 // write your code here
+switch Double(y) / x > Double(3) {
+case true:
+	print("true")
+	
+case false:
+	print("false")
+}
```
Comment (JimCampagno) [inline MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift]: Need to remove the answer.

File: MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift [hunk 4]
```diff
@@ -8,29 +8,129 @@
...
- */
+/*: Question 3
+### 3. Print "true" if y is greater than x and a divided by b is grater than 9
+*/
+// write your code here
+switch Double(y) > x  &&  a / b > 9 {
+case true:
+	print("true")
+case false:
+	print("false")
```
Comment (JimCampagno) [inline MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift]: Need to remove the answer.

File: MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift [hunk 5]
```diff
@@ -8,29 +8,129 @@
...
+### 4. Write a function "isGreater" that takes two Int arguments and returns "yep" if a is greater than b and "nope" if they're not
+*/
+// write your code here
+func isGreater(a:Int, b:Int) -> String {
+	switch a > b {
+	case true:
+		return "yep"
+	case false:
+		return "nope"
+	}
```
Comment (JimCampagno) [inline MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift]: Need to remove the answer.

File: MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift [hunk 6]
```diff
@@ -8,29 +8,129 @@
...
+	case "Yoda":
+		return true
+		
+	case "Vader":
+		return true
+		
+	default:
+		return false
+	}
+}
```
Comment (JimCampagno) [inline MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift]: Need to remove the answer.

File: MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift [hunk 7]
```diff
@@ -8,29 +8,129 @@
...
+	case 4...6:
+		print("medium")
+		
+	case 7...10:
+		print("big")
+		
+	default:
+		print("not sure")
+	}
+}
```
Comment (JimCampagno) [inline MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift]: Need to remove the answer.

**Comment by JimCampagno** [comment]
This is good :) 

Only thing that requires attention is the one comment having it state "is equal to zero" above and removing the various answers from the section where the student will write their code.

@guyardito

**Comment by JimCampagno** [comment]
The above stuff still needs to be done (if it's giving you some problems on your end), I can merge this in and make the change myself as they are small and don't require too much attention. 

Similar to my prior message.. if I don't hear from you, enjoy your 4th!!

@guyardito

**Comment by guyardito** [comment]
@JimCampagno   hmmm, I made and pushed those changes on June 28 @ 1:32 pm.

Should I push again?

**Comment by JimCampagno** [comment]
Yea, make the attempt to push again. If not, then I'll make the changes myself (no biggie).

@guyardito

**Comment by guyardito** [comment]
@JimCampagno 

done  (see trail below)

Guys-MBP:swift-switchLab-lab Guy$ git add .
Guys-MBP:swift-switchLab-lab Guy$ git commit -m "doing again b/c last time not getting recognized"
[master 0343d01] doing again b/c last time not getting recognized
 Committer: Guy Ardito Guy@Guys-MBP.home
Your name and email address were configured automatically based
on your username and hostname. Please check that they are accurate.
You can suppress this message by setting them explicitly. Run the
following command and follow the instructions in your editor to edit
your configuration file:

```
git config --global --edit
```

After doing this, you may fix the identity used for this commit with:

```
git commit --amend --reset-author
```

 3 files changed, 7 insertions(+), 71 deletions(-)
 rewrite MyPlayground.playground/playground.xcworkspace/xcuserdata/Guy.xcuserdatad/UserInterfaceState.xcuserstate (85%)
Guys-MBP:swift-switchLab-lab Guy$ git push
warning: push.default is unset; its implicit value has changed in
Git 2.0 from 'matching' to 'simple'. To squelch this message
and maintain the traditional behavior, use:

  git config --global push.default matching

To squelch this message and adopt the new behavior now, use:

  git config --global push.default simple

When push.default is set to 'matching', git will push local branches
to the remote branches that already exist with the same name.

Since Git 2.0, Git defaults to the more conservative 'simple'
behavior, which only pushes the current branch to the corresponding
remote branch that 'git pull' uses to update the current branch.

See 'git help config' and search for 'push.default' for further information.
(the 'simple' mode was introduced in Git 1.7.11. Use the similar mode
'current' instead of 'simple' if you sometimes use older versions of Git)

Counting objects: 11, done.
Delta compression using up to 8 threads.
Compressing objects: 100% (10/10), done.
Writing objects: 100% (11/11), 5.94 KiB | 0 bytes/s, done.
Total 11 (delta 3), reused 0 (delta 0)
To https://github.com/guyardito/swift-switchLab-lab.git
   7617d66..0343d01  master -> master
Guys-MBP:swift-switchLab-lab Guy$ date
Sat Jul  2 20:23:42 EDT 2016
Guys-MBP:swift-switchLab-lab Guy$

**Pull request merged**